moduleroot/.rubocop.yml.erb

```diff
@@ -6,8 +6,9 @@ defaults = {
     'rubocop-rspec',
   ],
   'AllCops' => {
+    'NewCops' => 'enable',
     'DisplayCopNames' => true,
-    'TargetRubyVersion' => '2.6',
+    'TargetRubyVersion' => '2.7',
```

Comment:
Why is that changed? It doesn't relate to the PR title

Reply:
we manually change the ruby version on PDK update everytime, so thought to cover as part of this as its small and quick one.
Let me know if I need to cover as part of diff PR?

Reply:
I think this needs to stay as ruby 2.6, to keep compatibility with the jruby versions still used in puppet server (IIRC)

Reply:
ahh i see, wasn't knowing that. Would like to know more about it. Thanks for pitching in @jordanbreen28.

Reply:
resolved, ready for review again.